learn/wavemaker-release-notes/v10-7-2.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "WaveMaker 10.7.2 - Release date: 21 June 2021"
+id: ""
+sidebar_label: "Release v10.7.2"
+---
+*WaveMaker releases an update with bug fixes.*
+
+## Improvements
+
+---
+
+- Improved the format of the **Currency** and **Number** widgets to support natural and financial modes by setting the **Input Mode** property.
+- For the **Upload widget**, added the ability to aptly highlight the dropzone when using the drag and drop approach.
```

Comment:
For the **Upload widget**, added the ability to aptly highlight the dropzone when the file is dropped inside the app. This highlighting can further be customized with the CSS theme applied on the project.